src/commands/publish.js

```diff
@@ -0,0 +1,215 @@
+const credentials = require('../config/credentials')
+const { default: Command, flags } = require('@oclif/command')
+const { default: chalk } = require('chalk')
+const manifest = require('../config/manifest')
+const api = require('../config/axios')
+const { firebase } = require('../config/firebase')
+const readline = require('readline')
+const moment = require('moment')
+const semver = require('semver')
+const { isYes } = require('../utils/index')
+class PublishCommand extends Command {
+  async run () {
+    try {
+      const { flags } = this.parse(PublishCommand)
+      if (flags.version && !semver.valid(flags.version)) {
+        this.error(`Version must be in x.x.x format`)
+        process.exit(0)
+      }
+
+      if (!this.commandSintaxeValid(flags)) {
+        this.error(`Use only one flag  --version --patch, --minor, --major`)
+        process.exit(0)
+      }
+
+      if (!manifest.exists()) {
+        this.warning('Please select your extension. Try run qt selectExtension')
```

Comment:
O comando é select-extension
```suggestion
        this.warning('Please select your extension. Try run qt select-extension')
```